pkg/config/config.go

```diff
@@ -86,6 +86,7 @@ type YAMLConfig struct {
 	Description          *string               `json:"description" yaml:"description"`
 	NebulaClientSettings *NebulaClientSettings `json:"clientSettings" yaml:"clientSettings"`
 	LogPath              *string               `json:"logPath" yaml:"logPath"`
+	CVSComma             *string               `json:"csvComma" yaml:"csvComma"`
```

Comment:
This is a csv option, why not move into the `CSVConfig` struct ? And I think the option's name `Delimiter` is better than `CVSComma`.

Reply:
you need think all csv file have one common format

Reply:
I don't think so. The meaning of designing CSVConfig is to let user configure each data file, and in the future, maybe there's some else XMLConfig or ExcelConfig to support excel or xml file. We shouldn't put the option of csv configuration in global scope.

Reply:
This is done, Config CSV comma for each file.I removed it